posthog/hogql/functions/mapping.py

```diff
@@ -831,6 +832,11 @@ def compare_types(arg_types: list[ConstantType], sig_arg_types: tuple[ConstantTy
     "nth_value": HogQLFunctionMeta("nth_value", 2, 2),
     "lagInFrame": HogQLFunctionMeta("lagInFrame", 1, 1),
     "leadInFrame": HogQLFunctionMeta("leadInFrame", 1, 1),
+    ## UDFS
+    "aggregate_funnel": HogQLFunctionMeta("aggregate_funnel", 6, 6, aggregate=False),
+    "aggregate_funnel_array": HogQLFunctionMeta("aggregate_funnel_array", 6, 6, aggregate=False),
+    "aggregate_funnel_cohort": HogQLFunctionMeta("aggregate_funnel_cohort", 6, 6, aggregate=False),
+    "test_function_python": HogQLFunctionMeta("test_function_python", 6, 6, aggregate=False),
```

Comment:
Looks like this function is unused.

Reply:
Thanks for the heads up. Yeah, we should remove it from the mapping so people can't use it.

I'll leave it in clickhouse because it's useful as a debugging function - it's the same as the other function, but it returns the output as a string, so if there are any parsing issues, it's easy to figure out.